providers/ldap/ldap_client.go

```diff
@@ -826,3 +837,20 @@ func (l *LClient) userRecord(search *ldap.SearchRequest, lConn *ldap.Conn) (mode
 	token.AccessToken = userIdentity.ExternalId
 	return token, status, nil
 }
+
+func (l *LClient) printMoreThanOneResult(result *ldap.SearchResult) {
+	for idx, e := range result.Entries {
+		buffer := bytes.Buffer{}
+		for _, v := range e.Attributes {
+			buffer.WriteString(v.Name)
+			buffer.WriteString(":[")
+			for i := 0; i < (len(v.Values) - 1); i++ {
+				buffer.WriteString(v.Values[i])
+				buffer.WriteString(" ")
+			}
+			buffer.WriteString(v.Values[len(v.Values)-1])
+			buffer.WriteString("] ")
+		}
+		log.Debugf("(GenerateToken) Query Result %v: DN: %v, Attributes: %v", idx, e.DN, buffer.String())
```

Comment:
generateToken was just one of the places it was called.

Reply:
oops! yes I'll change the message